orbs/utils.yml

```diff
@@ -347,3 +347,25 @@ commands:
                       wget https://github.com/jwilder/dockerize/releases/download/<<parameters.version>>/dockerize-linux-amd64-<<parameters.version>>.tar.gz
                       sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-<<parameters.version>>.tar.gz
                       rm dockerize-linux-amd64-<<parameters.version>>.tar.gz
+    verify_changelog:
```

Comment:
Maybe `require_changelog_between_tags` for a more specific name?

Reply:
Makes sense. I wasn't happy with that name.